modular_skyrat/modules/death_consequences_perk/death_consequences_customization.dm

```diff
@@ -189,40 +208,43 @@
 	minimum = 0
 	maximum = 150
 
+	config_name = "Crit threshold reduction: Maximum reduction"
+	config_desc = "When at the ending degradation percent, crit threshold will be reduced by this, \
+	with lower percentages causing equally displaced reducions, such as having 50% degradation causing 50% of this to be applied."
```

Comment:
For the future (because I'm too lazy to go over the entire file again), when doing multi-line strings, try to increment the indentation of the lines coming after the first one, so it's clearer that they're apart of a separate chunk of text, it helps at a quick glance when looking through the code.
```suggestion
		with lower percentages causing equally displaced reducions, such as having 50% degradation causing 50% of this to be applied."
```